### PR TITLE
[10.0] [ADD]  sale_promotion_rule: display discount amount on the sale order

### PR DESCRIPTION
**Pull request by lmignon**
- [ ] includes #755  

This addon allow you to display the amount of the discount applied by the sale promotion rule addon

### CODE REVIEW

**Review by Cedric-Pigeon — APPROVED** [review]
code review

File: sale_promotion_rule_display_discount_amount/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
```
Comment (rousseldenis) [inline sale_promotion_rule_display_discount_amount/__manifest__.py:5]: Could you add some missing entries and remove void ones ?

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

File: sale_promotion_rule_display_discount_amount/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "name": "Sale Promotion Rule Display Discount Amount",
+    "summary": """
+        This addon allows you to display the amount of the discount applied by
+        the sale promotion rules on the sale order""",
+    "version": "10.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow/",
+    "depends": ["sale_promotion_rule", "sale_discount_display_amount"],
+    "auto_isntall": "True",
```
Comment (rousseldenis) [inline sale_promotion_rule_display_discount_amount/__manifest__.py]: Fix this

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Minor comments

File: sale_promotion_rule_display_discount_amount/__manifest__.py (hunk 2 above)
Comment (lmignon) [reply sale_promotion_rule_display_discount_amount/__manifest__.py]: @rousseldenis Fix what?

Comment (rousseldenis) [reply sale_promotion_rule_display_discount_amount/__manifest__.py]: 'auto_install' and not 'auto_isntall' :smile:

Comment (lmignon) [reply sale_promotion_rule_display_discount_amount/__manifest__.py]: good catch @rousseldenis !

**Comment by lmignon** [comment]
@rousseldenis Changes done... Can we move forward? ping @sebastienbeau

**Comment by lmignon** [comment]
travis is :red_circle: due to https://github.com/OCA/sale-workflow/issues/817

**Comment by rousseldenis** [comment]
Rebased this one. Wait for travis.

**Pull request closed without merging**